install_scripts/instrument_upgrade.bat

```diff
@@ -0,0 +1,14 @@
+REM Upgrade NDXDEMO for sprint review
+
+set "SERVER_SOURCE=\\isis.cclrc.ac.uk\inst$\Kits$\CompGroup\ICP\EPICS\EPICS_win7_x64\BUILD-3125\EPICS"
+set "CLIENT_SOURCE=\\isis.cclrc.ac.uk\inst$\Kits$\CompGroup\ICP\Client\BUILD179"
+set "GENIE_PYTHON=\\isis.cclrc.ac.uk\inst$\Kits$\CompGroup\ICP\genie_python\BUILD-146"
```

Comment:
For an instrument update we probably just want a release directory as in `training_machine`